backends/rapidpro/status.go

```diff
@@ -140,6 +140,11 @@ func (b *backend) flushStatusFile(filename string, contents []byte) error {
 		return nil
 	}
 
+	// Ignore wrong status update for incoming messages
+	if err.Error() == "Incoming messages can only be PENDING or HANDLED" {
```

Comment:
This should be comparing against a constant like above not doing string comparison.